### PR TITLE
Emoji picker

### LINKED ISSUE #196: Emoji picker

### PR DESCRIPTION
**Pull request by szuperaz**
closes #196

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,13 +79,15 @@
     "@angular/platform-browser": "~12.2.0",
     "@angular/platform-browser-dynamic": "~12.2.0",
     "@angular/router": "~12.2.0",
+    "@ctrl/ngx-emoji-mart": "^6.2.0",
```
Comment (petyosi) [inline package.json:82]: If we don't provide this by default, why should we have it as a dependency?

**Review by petyosi — APPROVED** [review]
This looks good. The only thing I spotted was the dependency in package.json

File: package.json (hunk 1 above)
Comment (szuperaz) [reply package.json:82]: And the sample app has the emoji picker set up

Comment (petyosi) [reply package.json:82]: Got it

**Pull request merged**